packages/slider/package.json

```diff
@@ -23,8 +23,8 @@
     "styled-system": "^3.1.6"
   },
   "peerDependencies": {
-    "pcln-design-system": "^1.0.1",
-    "styled-components": "^3.4.9"
+    "pcln-design-system": ">=1.x || >=2.x",
+    "styled-components": ">=3.x || >= 4.x"
   },
```

Comment:
I think `jest-styled-components` should be removed from this package as well, since it's a dev dependency in the root

Reply:
Removed `jest-styled-components` as well from `package.json` of the slider package.